core/spec/models/mno_enterprise/user_spec.rb

```diff
@@ -385,6 +417,82 @@ def reload_user
       end
     end
 
+    describe '#requires_otp_for_login?' do
+      subject { user.requires_otp_for_login? }
+
+      context 'when user is an admin' do
+        let(:user) { build(:user, :admin) }
```

Comment:
not a huge deal, but we can refactor this so it's not necessary to redefine `user` over and over again.